scripts/feature_toggle_report_generator.py

```diff
@@ -294,16 +294,24 @@ def bool_for_null_numbers(n):
         if self.toggle_type == 'WaffleSwitch':
             return self._raw_state_data['active']
         elif self.toggle_type == 'WaffleFlag':
-            return (
-                self._raw_state_data['everyone'] or
-                bool_for_null_numbers(self._raw_state_data['percent']) or
-                self._raw_state_data['testing'] or
-                self._raw_state_data['superusers'] or
-                self._raw_state_data['staff'] or
-                self._raw_state_data['authenticated'] or
-                bool(self._raw_state_data['languages']) or
-                self._raw_state_data['rollout']
-            )
+            # the WaffleFlag option `everyone` overrides all other options.
+            # However, it must be explicitly set to Yes(True) or No(False)
+            # in the GUI. Otherwise, it is set to Unknown(None) and WaffleFlag
+            # defers to the other options to determine the state of the flag.
+            if self._raw_state_data['everyone']:
+                return True
+            elif self._raw_state_data['everyone'] is False:
+                return False
+            else:
+                return (
+                    bool_for_null_numbers(self._raw_state_data['percent']) or
+                    self._raw_state_data['testing'] or
+                    self._raw_state_data['superusers'] or
+                    self._raw_state_data['staff'] or
+                    self._raw_state_data['authenticated'] or
+                    bool(self._raw_state_data['languages']) or
+                    self._raw_state_data['rollout']
```

Comment:
1. This is missing users/groups.  I know we've used `users` in the past, but I have never seen us use `languages`.
2. I think you can drop `rollout`.  It affects cookie expiration, but not On/Off nature of the cookie.  I doubt it would be on with everything else off, so it isn't a big deal.

Reply:
Ok, good call. Will address in a subsequent PR.